docs/content/DropdownMenu.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: DropdownMenu
+---
+
+A `DropdownMenu` provides an anchor (button by default) that will open a floating menu of selectable items. The menu can be opened and navigated using keyboard or mouse. When an item is selected, the menu will close and the anchor contents will be updated with the selection.
+
+## Example
+
+```javascript live noinline
+function DemoComponent() {
+  const items = React.useMemo(() => [{text: '🔵 Cyan', id: 5}, {text: '🔴 Magenta'}, {text: '🟡 Yellow'}], [])
+  const [selectedItem, setSelectedItem] = React.useState()
+
+  return (
+    <DropdownMenu
+      renderAnchor={({children, 'aria-labelledby': ariaLabelledBy, ...anchorProps}) => (
+        <DropdownButton aria-labelledby={`favorite-color-label ${ariaLabelledBy}`} {...anchorProps}>
+          {children}
+        </DropdownButton>
```

Comment:
I believe consumers will have access to the dropdown button component as `DropdownMenu.Button` not `DropdownButton`, right?

```suggestion
        <DropdownMenu.Button aria-labelledby={`favorite-color-label ${ariaLabelledBy}`} {...anchorProps}>
          {children}
        </DropdownMenu.Button>
```

Reply:
`DropdownButton` could be used with both `DropdownMenu` and `ActionMenu`, right? If so, should it have a name/namespace that doesn’t match one of those exactly (i.e. that is more general)?

Reply:
Ah, I see. I didn't realize it was used with ActionMenu as well. In that case, I think leaving it as `DropdownButton` is fine 👍